commands-yml/commands/device/app/background-app.yml

```diff
@@ -0,0 +1,72 @@
+---
+name: Background App
+short_description: Send the currently running app for this session to the background
```

Comment:
We need a long description to discuss the differences in xctest-based iOS automation.

See:
* https://github.com/appium/appium/issues/7741 
* https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/general.js#L27-L70